packages/plugin-char/src/CharNode.ts

```diff
@@ -41,22 +41,6 @@ export class CharNode extends InlineNode {
     // Public
     //--------------------------------------------------------------------------
 
-    /**
-     * Return the length of this VNode.
-     */
-    get length(): number {
-        return 1;
-    }
-    /**
-     * Return this VNode's inner text (concatenation of all descendent
-     * char nodes values).
-     *
-     * @param __current
-     */
-    text(__current = ''): string {
```

Comment:
Why did you remove this? Was it moved somewhere else instead?

Reply:
You have quite the keen eye. 🦅 I removed it because it was not used anywhere in production code.